jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -76,6 +76,25 @@ teardown() {
     [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_Url}" = "jdbc:mariadb://myapp-host:3306/rhpam-mariadb?pinGlobalTxToPhysicalConnection=true\&amp;enabledSslProtocolSuites=TLSv1.2" ]
 }
 
+@test "test if the EJB_TIMER datasource has been auto-configured also with RHPAM_XA_CONNECTION_PROPERTY_Url and driver is mariadb" {
+    local expected_timer_service="EJB_TIMER"
+    local expected_datasources="EJB_TIMER,RHPAM"
+    local expected_url="jdbc:mariadb://myapp-host:3306/rhpam-mariadb?enabledSslProtocolSuites=TLSv1.2"
+    export DATASOURCES="RHPAM"
+    export RHPAM_DRIVER="mariadb"
+    export RHPAM_XA_CONNECTION_PROPERTY_Url="${expected_url}"
+    export RHPAM_NONXA="false"
+    configure_EJB_Timer_datasource >&2
+    echo "Expected EJB_TIMER url is ${EJB_TIMER_XA_CONNECTION_PROPERTY_Url}" >&2
+    echo "Expected RHPAM_URL is '${RHPAM_URL}'" >&2
+    echo "Expected RHPAM_XA_CONNECTION_PROPERTY_Url is ${RHPAM_XA_CONNECTION_PROPERTY_Url}" >&2
+    echo "Expected DATASOURCES is ${DATASOURCES}" >&2
+    [ "${TIMER_SERVICE_DATA_STORE^^}" = "${expected_timer_service}" ]
+    [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_Url}" = "jdbc:mariadb://myapp-host:3306/rhpam-mariadb?enabledSslProtocolSuites=TLSv1.2\&amp;pinGlobalTxToPhysicalConnection=true\&amp;enabledSslProtocolSuites=TLSv1.2" ]
+    [ "${RHPAM_XA_CONNECTION_PROPERTY_Url}" = "jdbc:mariadb://myapp-host:3306/rhpam-mariadb?enabledSslProtocolSuites=TLSv1.2" ]
```

Comment:
This is the expected value for RHPAM_URL, for XA both EJB and RHPAM Url must be the same.